tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -263,6 +267,69 @@ def compute_el_block_hash_for_block(spec, block):
         spec, block.body.execution_payload, block.parent_root, requests_hash)
 
 
+def build_empty_execution_payload_header(spec, state):
+    latest = state.latest_execution_payload_header
+    timestamp = spec.compute_timestamp_at_slot(state, state.slot)
+    execution_payload = build_empty_execution_payload(spec, state)
+
+    execution_payload_header = spec.ExecutionPayloadHeader(
+        parent_hash = latest.block_hash,
+        fee_recipient = spec.ExecutionAddress(),
+        state_root = latest.state_root,
+        receipts_root = spec.Bytes32(),
+        logs_bloom = spec.ByteVector[spec.BYTES_PER_LOGS_BLOOM](),
+        prev_randao = spec.Bytes32(),
+        block_number = 0,
+        gas_limit = 0,
+        gas_used = 0,
+        timestamp = timestamp,
+        extra_data = spec.ByteList[spec.MAX_EXTRA_DATA_BYTES](),
+        base_fee_per_gas = 0,
+        block_hash = compute_el_block_hash(spec, execution_payload, state),
+        transactions_root = spec.hash_tree_root(execution_payload.transactions),
+    )
+    if is_post_capella(spec):
+        execution_payload_header.withdrawals_root = spec.Root()
+
+    if is_post_deneb(spec):
+        execution_payload_header.blob_gas_used = 0
+        execution_payload_header.excess_blob_gas = 0
+
+    return execution_payload_header
+
+
+def build_randomized_execution_payload_header(spec, state, rng=Random(10000)):
+    execution_payload_header = build_empty_execution_payload_header(spec, state)
```

Comment:
Do we need an empty payload header here? Theoretically can be called from other places if needed. Or I can delete this function and build execution payload header right here

Reply:
Hey Katya. I would delete that function & builder the random exec payload right here 👍